v5/core/core.go

```diff
@@ -105,10 +107,30 @@ func (core *Core) Setup(sdkSettings model.SdkSettings, deviceProperties model.De
 		core.environment = client.NewSaasEnvironment(envApi)
 	}
 
-	// TODO Analytics.Analytics.Initialize(deviceProperties.RolloutKey, deviceProperties)
+	analyticsDisabled := (roxOptions != nil && roxOptions.IsAnalyticsReportingDisabled()) ||
+		roxyPath == "" ||
+		!core.internalFlags.IsEnabled("rox.internal.analytics")
```

Comment:
Can't check it here, it's part of client-data
Here, it's before we fetch the data
Unless I missed something, it's unlikely it will work as expected

Reply:
Well...yea. Also, client data can change. Now realizing that the sdk would need to be re-intialized if `rox.internal.analytics` were disabled initially and then enabled. So the analytics check should be in the impressions invoker. No biggie.

Reply:
Done.